src/en/help-azure.md

```diff
@@ -26,44 +27,53 @@ Before you can use Azure with Juju, you will need to import your Azure
 account credentials into Juju. Retrieving those credentials is easy, thanks to
 Microsoft's new [Azure Cloud Shell][azureshell].
 
-This can be installed using *[snap][snapcraft]*:
+Azure Cloud Shell can be installed using *[npm][npminfo]*. Enter the following
```

Comment:
"Azure Cloud Shell" is something else. That's the shell you can get in the Azure portal web UI. This is just the "Azure CLI".